src/Model/User.php

```diff
@@ -1113,7 +1113,7 @@ public static function getStatistics()
 			FROM `user`
 			INNER JOIN `profile` ON `profile`.`uid` = `user`.`uid`
 			INNER JOIN `contact` ON `contact`.`uid` = `user`.`uid` AND `contact`.`self`
-			WHERE (`profile`.`publish` OR `profile`.`net-publish`) AND `user`.`verified`
+			WHERE `user`.`verified` AND `user`.`login_date` > '0001-01-01' AND NOT `user`.`account-type` = 3
```

Comment:
Please use the PHP constants for the literal values you are using in the query.
```suggestion
			WHERE `user`.`verified` AND `user`.`login_date` > ? AND NOT `user`.`account-type` != ?
```
(Please note the operator switch for `` `user`.`account-type` ``, you were selecting only communities before)

And then you should add `, DBA::NULL_DATETIME, self::ACCOUNT_TYPE_COMMUNITY` in the parameter list of the `DBA::p` call after the query string.

Reply:
Should I add line breaks for each AND in the where condition and for both parameters? When I just add the parameters it looks weird.

Reply:
Looks better. I hope I do this right.